exonum-java-binding/core/src/main/java/com/exonum/binding/core/storage/indices/KeySetIndexProxy.java

```diff
@@ -220,6 +220,20 @@ public void remove(E e) {
     nativeRemove(getNativeHandle(), dbElement);
   }
 
+  /**
+   * Returns the number of entries in this set.
```

Comment:
The existing set docs refer to them as 'elements'. Maps, however, use 'entries'.